backend/src/main/java/com/google/backend/servlets/AlertsDataServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.backend.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.models.Alert;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets Alert information from the Datastore and returns the JSON. */
+@WebServlet("/api/v1/alerts-data")
+public class AlertsDataServlet extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    List<Alert> alertList = new ArrayList<>();
+
+    for (Entity entity : results.asIterable()) {
```

Comment:
optional: can use stream if you want to shorten this lol
```
import static java.util.stream.Collectors.toList;

List<Alert> alertList = results.asIterable().stream()
    .map(entity -> Alert.createAlertFromEntity(entity))
    .collect(toList());
```
something like this should prob work, i personally just really like using streams hahah

Reply:
for additional read: Effective Java 3rd edition Item 45  "Use streams judiciously", and https://docs.google.com/document/d/141wpk6X6et6Ym2qKWgY3V2u4dJNYB2hnvJYp6UJkqIc/edit?hl=en#heading=h.u1btcr9i6q5o on when to use streams vs not streams, etc.